src/aiforemast.py

```diff
@@ -56,9 +56,12 @@ def cacheModels(modelHolder, max_cache_size = MAX_CACHE_SIZE):
         jobId = jobs[MAX_CACHE_SIZE-1]
         cachedJobs.pop(jobId)
         jobs.remove(jobId)
-    uuid = modelHolder.id
-    cachedJobs[uuid]= modelHolder
-    jobs.append(uuid) 
+    uuid = modelHolder.getId()
```

Comment:
This line will no longer work because id was moved to a property and `getId` no longer exists. The conditional can also be simplified a bit too, although I don't understand the surrounding context of the code quite enough to say whether that is a benefit to readability.

```python
uuid = modelHolder.id
if uuid not in cachedJobs:
    jobs.append(uuid)
cachedJobs[uuid] = modelHolder
```